types/glob/index.d.ts

```diff
@@ -4,9 +4,13 @@
 //                 voy <https://github.com/voy>
 //                 Klaus Meinhardt <https://github.com/ajafff>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9
```

Comment:
why is this change necessary?

Reply:
I'm not sure. I believe this happened when importing Stat 🤔 

Take a look at the previous CI: https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/508680421

Reply:
@ajafff I just changed the import so it works with all TS Versions again 👍